### DISCUSSION TITLE
my provider of AutoDisposeFutureProvider  don't implement when I push to its screen again

### DISCUSSION THREAD

**Discussion by Abouassi98** [rrousselGit/riverpod#1676] [Q&A]
![image](https://user-images.githubusercontent.com/53384802/191904399-01eec473-6af7-4f8a-9c60-addc335f8b45.png)
![image](https://user-images.githubusercontent.com/53384802/191901318-9916672b-a64f-4dc3-a78d-02b4c9bfd243.png)
and this dispose function doesn't implement at all 

environment:
  sdk: '>=2.18.1 <3.0.0

hooks_riverpod: ^2.0.0-dev.9
flutter :3.3.2

**Comment by rrousselGit** [comment]
Hello

I don't understand your screenshots. Could you clarify?
